src/reducers/engine.ts

```diff
@@ -40,6 +40,7 @@ export const types = {
   UPDATE_IS_LIVE: `${component}/UPDATE_IS_LIVE`,
   UPDATE_IS_DVR: `${component}/UPDATE_IS_DVR`,
   UPDATE_IS_IMG: `${component}/UPDATE_IS_IMG`,
+  UPDATE_IS_DOC: `${component}/UPDATE_IS_DOC`,
```

Comment:
not critical, but it would be nice to update this to document as well, because it updates the value of isDocument field

Reply:
I kept it aligned with other reducers keys like `UPDATE_IS_IMG` where source named `image`, etc